core/templates/css/oppia.css

```diff
@@ -810,7 +810,8 @@ textarea {
   Styles for footer
 */
 
-.oppia-footer-container h4 {
+.oppia-footer-container h3 {
+  color: white;
```

Comment:
Please prefer HEX codes for colors (see [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#css)).